### PR TITLE
fix  typos reseting -> resetting

### PR DESCRIPTION
**Pull request by yeya24**
fix a typos reseting -> resetting   in line 656 and 606

**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
fix a typo

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: test/e2e/autoscaling/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -653,7 +653,7 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		ReserveMemory(f, "memory-reservation", extraPods, totalMemoryReservation, false, defaultTimeout)
 
 		// Apparently GKE master is restarted couple minutes after the node pool is added
-		// reseting all the timers in scale down code. Adding 5 extra minutes to workaround
+		// resetting all the timers in scale down code. Adding 5 extra minutes to workaround
```
Comment (nikhita) [inline test/e2e/autoscaling/cluster_size_autoscaling.go]: Can you also fix this same typo that is occuring in two other places: https://github.com/kubernetes/kubernetes/search?q=reseting&unscoped_q=reseting?

Comment (yeya24) [reply test/e2e/autoscaling/cluster_size_autoscaling.go]: fix it. plz review it

**Comment by oomichi** [comment]
Still pkg/kubelet/apis/deviceplugin/v1beta1/api.pb.go contains the same typo.

**Comment by nikhita** [comment]
@yeya24 can you squash the commits into a single commit? Looks good otherwise :)

**Comment by yeya24** [comment]
> @yeya24 can you squash the commits into a single commit? Looks good otherwise :)

finish it! @nikhita

**Comment by nikhita** [comment]
/assign @jszczepkowski @random-liu
for approval

**Pull request closed without merging**